cmd/process-agent/main_common.go

```diff
@@ -230,7 +225,7 @@ func runAgent(globalParams *command.GlobalParams, exit chan struct{}) {
 	if globalParams.Info {
 		// using the debug port to get info to work
 		url := fmt.Sprintf("http://localhost:%d/debug/vars", expVarPort)
-		if err := Info(os.Stdout, cfg, url); err != nil {
+		if err := Info(os.Stdout, url); err != nil {
```

Comment:
Is the `--info` command still showing the hostname now that `initInfo` is called after Info()?

Reply:
Good question, let me verify again. Thank you.

Reply:
You were absolutely right @mbotarro, fixed it in [9813017](https://github.com/DataDog/datadog-agent/pull/14885/commits/9813017c61c7ca5e74ff083c454a2b72ec9cdbdd)

Reply:
Thanks for the fix!!